consensus/hotstuff/timeoutcollector/timeout_collector.go

```diff
@@ -77,6 +86,7 @@ func (c *TimeoutCollector) processTimeout(timeout *model.TimeoutObject) error {
 	err := c.processor.Process(timeout)
 	if err != nil {
 		if model.IsInvalidTimeoutError(err) {
+			c.log.Err(err).Msgf("invalid timeout detected")
```

Comment:
Right, and we have that already, this will be logged as warning by the consumer of events but for know I think it's useful to have a message that specifically prints the error for us to get better observability if something goes wrong. During previous tests this type of logs proved to be useful.